tests/oscap_string/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+add_executable(test_oscap_string
+	"test_oscap_string.c"
+	${CMAKE_SOURCE_DIR}/src/common/oscap_string.c
+	${CMAKE_SOURCE_DIR}/src/common/oscap_buffer.c
+	${CMAKE_SOURCE_DIR}/src/common/alloc.c
```

Comment:
we are linking to openscap so why is this needed? because those are private symbols? that should be commented here.

Reply:
@mpreisler Yes, that's because some of the tested functions are private. I'll provide a comment.